mso/resource_mso_site.go

```diff
@@ -113,49 +113,87 @@ func resourceMSOSite() *schema.Resource {
 func resourceMSOSiteImport(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
 	log.Printf("[DEBUG] Site: Beginning Import")
 	msoClient := m.(*client.Client)
-	con, err := msoClient.GetViaURL("api/v1/sites" + d.Id())
+
+	dn := d.Id()
+
+	var apiVersion string
+	platform := msoClient.GetPlatform()
+	if platform == "nd" {
+		apiVersion = "v2"
+	} else {
+		apiVersion = "v1"
+	}
+	path := fmt.Sprintf("api/%v/sites/%v", apiVersion, dn)
+
+	con, err := msoClient.GetViaURL(path)
 	if err != nil {
 		return nil, err
 	}
 
-	d.SetId(models.StripQuotes(con.S("id").String()))
-
-	d.Set("name", models.StripQuotes(con.S("name").String()))
+	if platform == "nd" {
+		dataConAttr := con.S("common")
+		d.SetId(models.StripQuotes(con.S("id").String()))
```

Comment:
Shouldn't this be outside of the if ? Right now it does not seems to apply to non-ND ones.